visvis2/renderers/_wgpu.py

```diff
@@ -95,7 +121,7 @@ def compose_pipeline(self, wobject):
             ctypes.memmove(buffer.mapping, array.ctypes.data, nbytes)
             buffer.unmap()
             shader_location = len(buffers)
-            buffers[shader_location] = buffer
+            # buffers[shader_location] = buffer
             vbo_des = {
                 "array_stride": 3 * 4,
                 "stepmode": wgpu.InputStepMode.vertex,
```

Comment:
I see this enum also has the option "instance", do you know what that is?

Reply:
Instance rendering. So in that case the vertex does not change per-vertex, but per instance, I suppose :)  We should look into that once we are in a better shape. I think instance rendering can be very useful indeed.